install-local.sh

```diff
@@ -1,34 +1,40 @@
 usage() {
-    echo
-    echo "nsuite installer options:"
-    echo
-    echo "   arbor       : build Arbor"
-    echo "   neuron      : build NEURON"
-    echo "   coreneuron  : build CoreNEURON"
-    echo "   all         : build all simulators"
-    echo "   -e filename : source filename before building"
-    echo
-    echo "examples:"
-    echo
-    echo "install only Arbor:"
-    echo "$ install arbor"
-    echo
-    echo "install Arbor, NEURON and CoreNEURON:"
-    echo "$ install all"
-    echo
-    echo "install NEURON using environment configured in config.sh:"
-    echo "$ install neuron -e config.sh"
-    echo
+    cat <<'_end_'
+Usage: install-local.sh [-e SCRIPT] [--prefix=PATH] TARGET
+
+Setup NSuite framework, build and install simulators, benchmarks,
+and validation tests.
+
+Options:
+    -e SCRIPT       Source SCRIPT before building.
+    --prefix=PATH   Use PATH as base for install and other run-time
```

Comment:
Can we just go with `--prefix PATH`, not handle both `--prefix=X` and `--prefix X`? to make things simpler.

And maybe replace `-e SCRIPT` with `--env SCRIPT`, to keep consistent command line argument formatting?

My preference would be:
```
    --env SCRIPT
    --prefix PATH
```

Reply:
I've changed `-e` to `--env`. Per slack discussion, I've left both option styles (--long=FOO and --long FOO) for consistency with other software, and used '=' in the help text, following what GNU tools seem to generally do.